.envrc

```diff
@@ -38,7 +38,7 @@ export PGUSER=postgres
 PATH_add ./bin
 
 export CEDAR_ENV=impl
-export CEDAR_API_URL="https://webmethods-apigw.cedarimpl.cms.gov/gateway/Cedar%20Applications/1.1"
+export CEDAR_API_URL="webmethods-apigw.cedardev.cms.gov"
```

Comment:
will this always still get the correct api without the `/gateway/Cedar%20Applications/1.1` section?

Reply:
It will still grab it from here: https://github.com/CMSgov/easi-app/blob/master/pkg/cedar/swagger-impl.json#L9 and here https://github.com/CMSgov/easi-app/blob/master/pkg/cedar/translated_client.go#L21. It wasn't using this env variable for anything previously.